### DISCUSSION TITLE
Problem when forking go shared library.

### DISCUSSION THREAD

**Discussion by jclab-joseph** [Nitrokey/nethsm-pkcs11#6] [Q&A]
https://github.com/golang/go/issues/16855

Looking at the above issue, when a fork after loads a shared library made go, it crashes.

Is this project safe for this issue?

**Comment by ansiwen** [comment]
Good question. Probably not. We only used it with nginx so far, and that works without problem. Also, I'm not sure this issue is still valid. At least on ELF systems its possible to load several Go runtimes.
